internal/driver/testdata/pprof.cpu.cum.lines.text.show

```diff
@@ -1,3 +1,4 @@
+Using only nodes matching [12]00
```

Comment:
To make the output and the legend simpler and more compact, would it be better to just do something like:

Active filters: focus=W, ignore=X, show=Y, hide=Z

?